Jamfile

```diff
@@ -423,7 +423,7 @@ rule openssl-include-path ( properties * )
 		# on macOS, default to pick up openssl from the homebrew installation
 		# brew install openssl
 		# homebrew on M1 Macs install to /opt/homebrew
-		OPENSSL_INCLUDE = /opt/homebrew/opt/openssl/include /usr/local/opt/openssl/include ;
+		OPENSSL_INCLUDE = /opt/homebrew/opt/openssl@1.1/include /opt/homebrew/opt/openssl/include /usr/local/opt/openssl/include ;
```

Comment:
here too, you can specify this path on the `b2` command line: `openssl-include=/opt/homebrew/opt/openssl@1.1/include`